### PR TITLE
Fixing linter errors (no-verify)

### PR DESCRIPTION
**Pull request by awalshy**
## Fixing linter errors
----

## Details

- Added function return types
- Adding lint ignore for tests in js

Merging to actual branch : 
```
git fetch
git merge develop
```

### CODE REVIEW

**Comment by awalshy** [comment]
@Loriage il y a un test sur logger qui ne passe pas, je peux merge quand même ?

File: pcap/src/index.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function init() {
   })
 }
 
-(() => {
+;((): void => {
```
Comment (Psyroo) [inline pcap/src/index.ts:39]: not readable

File: msg_broker/src/index.test.ts [hunk 2]
```diff
@@ -3,13 +3,13 @@ const mock = { info: jest.fn() } as Record<keyof Logger, jest.Mock>
 jest.mock('../../utils/Logger/Logger', () => ({
   __esModule: true,
   Logger: class LoggerMock {
-    info = mock.info;
+    info = mock.info
   },
 }))
 
-
 import { main } from './index'
```
Comment (Psyroo) [inline msg_broker/src/index.test.ts:10]: imports should be at the top

File: msg_broker/src/index.test.ts [hunk 3]
```diff
@@ -3,13 +3,13 @@ const mock = { info: jest.fn() } as Record<keyof Logger, jest.Mock>
...
 
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```
Comment (Psyroo) [inline msg_broker/src/index.test.ts:12]: bof bof (english)

File: msg_broker/src/index.test.ts [hunk 4]
```diff
@@ -45,3 +45,4 @@ test('should read message from queue successful', async () => {
   expect(mock.info).toHaveBeenCalledWith('[PCAP] -', 'Message-01')
   expect(mock.info).toHaveBeenCalledWith('[DHCP] -', 'Message-01')
 })
+/* eslint-enable @typescript-eslint/explicit-function-return-type */
```
Comment (Psyroo) [inline msg_broker/src/index.test.ts:48]: bof bof (english)

File: utils/Logger/Logger.test.ts [hunk 5]
```diff
@@ -8,19 +8,22 @@ const logger = {
   warn: jest.fn(),
 } as Record<keyof Console, jest.Mock>
 
+/* eslint-disable @typescript-eslint/no-empty-function */
```
Comment (Psyroo) [inline utils/Logger/Logger.test.ts:11]: bof bof (english)

File: utils/Logger/Logger.test.ts [hunk 6]
```diff
@@ -8,19 +8,22 @@ const logger = {
...
     constructor() {}
 
-    warn = logger.warn;
-    info = logger.info;
-  }
+    warn = logger.warn
+    info = logger.info
+  },
 }))
+/* eslint-enable @typescript-eslint/no-empty-function */
```
Comment (Psyroo) [inline utils/Logger/Logger.test.ts:21]: bof bof (english)

File: pcap/src/index.ts (hunk 1 above)
Comment (awalshy) [reply pcap/src/index.ts:39]: @Loriage

File: msg_broker/src/index.test.ts (hunk 2 above)
Comment (awalshy) [reply msg_broker/src/index.test.ts:10]: @Loriage

File: msg_broker/src/index.test.ts (hunk 3 above)
Comment (awalshy) [reply msg_broker/src/index.test.ts:12]: It is a typo defined by eslint https://eslint.org/docs/2.13.1/user-guide/configuring

File: msg_broker/src/index.test.ts (hunk 4 above)
Comment (awalshy) [reply msg_broker/src/index.test.ts:48]: It is a typo defined by eslint https://eslint.org/docs/2.13.1/user-guide/configuring

File: utils/Logger/Logger.test.ts (hunk 6 above)
Comment (awalshy) [reply utils/Logger/Logger.test.ts:21]: It is a typo defined by eslint https://eslint.org/docs/2.13.1/user-guide/configuring

**Pull request merged**